app.conf.ctmpl

```diff
@@ -0,0 +1,12 @@
+{{with $environment := env "ENVIRONMENT"}}
+{{with $client := vault (printf "secret/dsde/avram/%s/oauth_client" $environment)}}
+{{with $googleProject := env "GOOGLE_PROJ"}}
+{{with $serviceVersion := env "SERVICE_VERSION"}}
+
+avram {
+    serviceVersion = "{{$serviceVersion}}"
```

Comment:
Just stuck configy-type stuff in here to make sure pulling from vault and reading environment variables works - these aren't the final things I want to put in here. I'm wondering what else we need in config right now? I assume a db config?

Reply:
Yeah probably need a slick/postgres config like: http://slick.lightbend.com/doc/3.1.0/database.html